tensorflow/compiler/xla/service/gpu/instruction_fusion.cc

```diff
@@ -87,7 +87,7 @@ bool GpuInstructionFusion::ShouldFuse(HloInstruction* consumer,
   auto producer = consumer->operand(operand_index);
 
   // The following checks are potentially expensive.
-  if (FusionWouldBeTooLarge(*consumer, *producer)) {
+  if (FusionWouldBeTooLarge(*consumer, *producer, true)) {
```

Comment:
nit: maybe write
`/*is_consumer_producer_fusion=*/true`
to annotate what the true means.

Reply:
Done.